rocky/rocky/templates/oois/ooi_list.html

```diff
@@ -37,13 +37,69 @@ <h1>
                     {% blocktranslate with length=ooi_list|length total=total_oois %}Showing {{ length }} of {{ total }} objects{% endblocktranslate %}
                 </p>
                 <form method="post"
-                      class="inline layout-wide checkboxes_required"
-                      action="#bulk_clearance_level_submit">
+                      id="ooi_list_form"
+                      class="inline layout-wide checkboxes_required">
                     {% csrf_token %}
                     <div class="horizontal-scroll">
-                        {% include "partials/form/form_errors.html" with form=select_oois_form %}
+                        <table class="nowrap">
+                            <caption class="visually-hidden">
+                              {% translate "Objects" %}
+                              <span class="visually-hidden">
+                                , {% translate "column headers with buttons are sortable" %}
```

Comment:
@ppvg is this comma intentionally?

Reply:
Yes, to provide syntactic separation, otherwise the caption as read to screen reader users would be an uninterrupted "objects column headers with button are sortable".

Reply:
I also see the formatter is unhappy with the code I proposed, sorry about that! :sweat_smile: